sound/soc/intel/boards/sof_sdw_rt_sdca_jack_common.c

```diff
@@ -88,6 +88,15 @@ static const char * const jack_codecs[] = {
 	"rt711", "rt712", "rt713", "rt722"
 };
 
+/*
+ * The sdca suffix is required for rt711 since there are two generations of the same chip.
+ * RT713 is an SDCA device but the sdca suffix is required for backwards-compatibility with
+ * previous UCM definitions.
+ */
+static const char * const need_sdca_suffix[] = {
+	"rt711", "rt713"
```

Comment:
there is rt715-sdca in UCM/ We fabricate that in some other place (from rt714), right? It is not a jack codec?

Reply:
> there is rt715-sdca in UCM. We fabricate that in some other place (from rt714), right? It is not a jack codec?

Right rt715-sdca is a dmic codec.